src/libinjector/libinjector.h

```diff
@@ -190,7 +190,8 @@ int injector_start_app(drakvuf_t drakvuf,
                        const char* binary_path,     // if -m = doppelganging
                        const char* target_process,  // if -m = doppelganging
                        bool break_loop_on_detection,
-                       injector_t* injector_to_be_freed); // out: iff break_loop_on_detection is set
+                       injector_t* injector_to_be_freed,
+                       bool global_search); // out: iff break_loop_on_detection is set
```

Comment:
The comment on the last line is now describing the wrong variable. Please keep the variable marked with `out` as the last one of the function.